src/components/AddQuestionTo.tsx

```diff
@@ -12,35 +12,30 @@ import { AirDBContext } from '../context/AirDBContext'
 import { Check, Close } from '@material-ui/icons'
 
 export default (props: any) => {
-  const { tests, questions, updateAirDB } = React.useContext(AirDBContext)
+  const { tests, questions, updateTestQuestions } = React.useContext(
+    AirDBContext
+  )
   const test = tests.filter(test => test.fields.ID === props.id)[0] || {
     fields: { title: '', questions: '' }
   }
   const [questionIDs, setQuestions] = React.useState<string[]>(
     (test.fields.questions || '').split(', ')
   )
 
-  const handleChange = useCallback(
-    (question: any) => {
-      setQuestions(prev => {
-        if (!prev.includes(JSON.stringify(question.fields.ID))) {
-          return [...prev, JSON.stringify(question.fields.ID)]
-        } else {
-          return prev.filter(
-            (id: string) => id !== JSON.stringify(question.fields.ID)
-          )
-        }
-      })
-    },
-    [questions]
-  )
+  const handleChange = useCallback((question: any) => {
+    setQuestions(prev => {
+      if (!prev.includes(JSON.stringify(question.fields.ID))) {
+        return [...prev, JSON.stringify(question.fields.ID)]
```

Comment:
Why are you stringify'ing these values? Do you have a sample of what it looks like?

Typically, if you absolutely have to do this, you'd want to only call `JSON.stringify` once and pass the variable around, but at the same time it's usually a code smell to compare with a stringify'ed result.

Reply:
I actually delete this file later on. It's just because I'm not sure how to make the number become a string. How might I better do this?